content/en/docs/aws/rds.md

```diff
@@ -0,0 +1,95 @@
+
+### Deploy Amazon RDS MySQL in your environment
+
+Before deploying MySQL database using Amazon RDS, let's get configuration parameters that are needed such as VpcId, SubnetIds and SecurityGroupId
+
+```shell
+# Use these commands to find VpcId, SubnetId and SecurityGroupId if you deployed your EKS cluster using eksctl
+# For DIY Kubernetes on AWS, modify tag name or values to get desired results
+export AWS_CLUSTER_NAME=<YOUR EKS CLUSTER NAME>
+
+# Below command will retrieve your VpcId
+aws ec2 describe-vpcs --filters Name=tag:alpha.eksctl.io/cluster-name,Values=$AWS_CLUSTER_NAME | jq -r '.Vpcs[].VpcId'
+
+# Below command will retrieve list of Private subnets
+# You need to use at least two Subnets from your List
+aws ec2 describe-subnets --filters Name=tag:alpha.eksctl.io/cluster-name,Values=$AWS_CLUSTER_NAME Name=tag:aws:cloudformation:logical-id,Values=SubnetPrivate* | jq -r '.Subnets[].SubnetId'
+
+# Below command will retrieve SecurityGroupId for your Worker nodes
+# This assumes all your Worker nodes share same SecurityGroups
+INSTANCE_ID=$(aws ec2 describe-instances --query 'Reservations[*].Instances[*].InstanceId' --filters "Name=tag-key,Values=eks:cluster-name" "Name=tag-value,Values=$AWS_CLUSTER_NAME" --output text)
+for i in "${INSTANCE_IDS[@]}"
+do
+  echo "SecurityGroup for EC2 instance $i ..."
+aws ec2 describe-instances --instance-ids $i | jq -r '.Reservations[].Instances[].SecurityGroups[].GroupId'
+done  
+```
+You can either use console or use attached CloudFormation template to deploy Amazon RDS database
+
+{{% alert title="Warning" color="warning" %}}
+The CloudFormation template deploys Amazon RDS for MySQL that is intended for Dev/Test environment.
+We highly recommend deploying Multi-AZ database for Production. Please review RDS [documentation](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Welcome.html) to learn more
+{{% /alert %}}
+
+[{{<figure src="/docs/images/aws/cloudformation-launch-stack.png">}}](https://console.aws.amazon.com/cloudformation/home?#/stacks/new?stackName=kubeflow-db&templateURL=https://cloudformation-kubeflow.s3-us-west-2.amazonaws.com/rds.yaml)
+
+Remember to select correct **AWS Region** for your cluster and click Next. We recommend you to change the **DBPassword**, if not it will dafault to `Kubefl0w`. Select VpcId, Subnets and SecurityGroupId before clicking Next. Take rest all defaults and click **Create Stack**.
+
+Once the CloudFormation is completed, click on Outputs tab to get RDS endpoint. If you didn't use CloudFormation, you can retrieve RDS endpoint through AWS management console for RDS on the Connectivity & security tab under Endpoint & port section. We will use it in next step while installing Kubeflow   
```

Comment:
Minor suggestion: 
1. please add a period(.) at the end of sentence.
2. next step -> the next step

Reply:
resolved